tecken/base/symboldownloader.py

```diff
@@ -22,16 +22,6 @@
 metrics = markus.get_metrics("tecken")
 
 
-class SymbolNotFound(Exception):
-    """Happens when you try to download a symbols file that doesn't exist"""
-
-
-class SymbolDownloadError(Exception):
-    def __init__(self, status_code, url):
-        self.status_code = status_code
-        self.url = url
```

Comment:
I split the symbolication API out into a separate service (Eliot). This is vestiges from the old symbolication code.